lib/process-services-cloud/src/lib/process/process-filters/components/edit-process-filter-cloud.component.ts

```diff
@@ -95,6 +95,7 @@ export class EditProcessFilterCloudComponent implements OnInit, OnChanges, OnDes
     action = new EventEmitter<ProcessFilterAction>();
 
     private _filter: ProcessFilterCloudModel;
+    protected formHasBeenChanged = false;
```

Comment:
does this boolean indicate that a form has changed or a filter? I guess a filter, if that's the case can we rename it to filterHasBeenChanged?